# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/ipc/VRLayerChild.h
# Commit: e1b0906509ef
# Full Hash: e1b0906509efa6433978b9f980ec639b7eafbc6d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix non-webgl CI tests. r=handyman
#   
#   * Revert some partial webgl+oop+vr code.
#   * More missing FuncScope.
#   * Fix compile errors.
# ==============================================================================

diff -r 7e2a2b1b416f -r e1b0906509ef gfx/vr/ipc/VRLayerChild.h
--- a/gfx/vr/ipc/VRLayerChild.h	Wed Jan 08 22:19:23 2020 +0000
+++ b/gfx/vr/ipc/VRLayerChild.h	Wed Jan 08 22:19:26 2020 +0000
@@ -61,6 +61,9 @@
   gfx::Rect mLeftEyeRect;
   gfx::Rect mRightEyeRect;
 
+  RefPtr<layers::SharedSurfaceTextureClient> mThisFrameTexture;
+  RefPtr<layers::SharedSurfaceTextureClient> mLastFrameTexture;
+
   uint64_t mLastSubmittedFrameId;
 };
 